docs/markdown/Builtin-options.md

```diff
@@ -82,6 +82,7 @@ Using the option as-is with no prefix affects all machines. For example:
 | stdsplit                             | true          | Split stdout and stderr in test logs                           | no             |
 | strip                                | false         | Strip targets on install                                       | no             |
 | unity {on, off, subprojects}         | off           | Unity build                                                    | no             |
+| unity_size {>=2}                     | 4             | Unity file block size                                          | no             |
```

Comment:
Shouldn't this be `unity_size {>=1} `?

Reply:
Unity size of 1 is equivalent to turning unity off altogether.

Reply:
Sure, but then the `(1, None, 4)` tuple in coredata should also be updated to reflect the documentation.

Reply:
D'oh! Fixed.